### ISSUE TITLE
AVT 1 - DatePicker Single w/Calendar & Fully Controlled have DAP violations for form control label

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4181]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
1. Go to the React DatePicker Single w/Calendar & Fully Controlled
2. Run DAP
3. Select a Date
4. Refresh the page to clear the date and the following issue is found on the copy button (see screenshot):
<img width="1371" alt="single-calender" src="https://user-images.githubusercontent.com/21676914/66075698-cc8d3b00-e521-11e9-99f6-fcfb604b287b.png">
<img width="1339" alt="Screen Shot 2019-10-02 at 2 28 33 PM" src="https://user-images.githubusercontent.com/21676914/66075706-ceef9500-e521-11e9-884e-8661ab9c6ef3.png">

**Comment by asudoh** [comment]
The DAP error cannot be seen at neither of stories with latest `master`, at http://localhost:9000/iframe.html?id=datepicker--single-with-calendar and http://localhost:9000/iframe.html?id=datepicker--fully-controlled, with November 2019 ruleset.
